framework/src/application/node/account.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export const accountAssetSchema = {
```

Comment:
This file contains all the types, would be nice to move it under `types` directory?

Reply:
types is standard to define typescript types, and schema is i think different